common/store.go

```diff
@@ -248,7 +251,10 @@ func createSchemaMysql(dbStore *DbStore, force bool) error {
 
 	for i := range entities {
 		entity := entities[i]
-		dbStore.Db.CreateTable(entity)
+		db := dbStore.Db.CreateTable(entity)
```

Comment:
it was a silent error, its just being made prominent here, but will check if we support rollback, else will create a card for it.